components/ws-daemon/leeway.Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN apk add --no-cache curl file \
 
 FROM ubuntu:22.04
 
+# trigger manual rebuild increasing the value
+ENV TRIGGER_REBUILD=1
```

Comment:
Are there any guidelines that when we should bump this value?

Reply:
Only when there are no real changes in the images, but we need to force a rebuild

Reply:
FYI, we use similar strategy in other repos, like `workspace-images`, to [trigger rebuilds to update ca-certs](https://github.com/gitpod-io/workspace-images/blob/17c7abc9942db32c41535d016e343c70f6f431ec/chunks/dep-cacert-update/Dockerfile#L7).